### PR TITLE
migrate monitoring to vue (new)

### PR DESCRIPTION
**Pull request by PhilBastian**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -100,3 +100,4 @@ src/scaffolding.config
 
 # Visual Studio Code
 .vscode
+src/ServicePulse.Host/angular/package-lock.json
```
Comment (mikeminutillo) [inline .gitignore:103]: Why are we ignoring this package lock and not the Vue one?

File: src/ServicePulse.Host/vue/package.json [hunk 2]
```diff
@@ -17,16 +22,29 @@
...
+    "@storybook/addon-actions": "^7.6.2",
+    "@storybook/addon-essentials": "^7.5.3",
+    "@storybook/addon-interactions": "^7.6.5",
+    "@storybook/addon-links": "^7.5.3",
+    "@storybook/blocks": "^7.5.3",
+    "@storybook/jest": "^0.2.3",
+    "@storybook/test-runner": "^0.14.1",
+    "@storybook/testing-library": "^0.2.2",
+    "@storybook/vue3": "^7.5.3",
+    "@storybook/vue3-vite": "^7.5.3",
```
Comment (mikeminutillo) [inline src/ServicePulse.Host/vue/package.json:45]: Should these be in here? We had a separate PR to remove the testing and were planning on re-introducing it.

File: src/ServicePulse.Host/vue/.eslintrc.cjs [hunk 3]
```diff
@@ -8,7 +8,7 @@ module.exports = {
     es6: true,
   },
   ignorePatterns: ["node_modules/**/*", "dist/**/*", "*.ts"],
-  extends: ["plugin:vue/vue3-essential", "eslint:recommended", "plugin:@typescript-eslint/recommended", "@vue/eslint-config-prettier/skip-formatting"],
+  extends: ["plugin:vue/vue3-essential", "eslint:recommended", "plugin:@typescript-eslint/recommended", "@vue/eslint-config-prettier/skip-formatting", "plugin:storybook/recommended"],
```
Comment (mikeminutillo) [inline src/ServicePulse.Host/vue/.eslintrc.cjs:11]: Another storybook thing. Not sure if it should be present in this PR

File: src/ServicePulse.Host/vue/src/composables/formatter.ts [hunk 4]
```diff
@@ -8,21 +8,20 @@ const dayDuration = moment.duration(24 * 60 * 60 * 1000);
 export function useFormatTime(value: string) {
   const duration = moment.duration(value);
   const time = { value: "0", unit: "" };
-  const asUtc = moment.utc(duration.asMilliseconds());
 
   if (duration >= dayDuration) {
-    time.value = asUtc.format("D [d] h [hr]");
+    time.value = formatTimeValue(duration.days()) + " d " + formatTimeValue(duration.hours()) + " hrs";
```
Comment (mikeminutillo) [inline src/ServicePulse.Host/vue/src/composables/formatter.ts:13]: Will this result in `2 days 1 hrs`? Does the original do that?

File: src/ServicePulse.Host/vue/src/composables/serviceLicense.ts [hunk 5]
```diff
@@ -8,13 +8,13 @@ import { LicenseStatus } from "@/resources/LicenseInfo";
...
 const trialExpiring =
-  '<div class="license-warning"><strong>Non-production development license expiring</strong><div>Your non-production development license will expire soon. To continue using the Particular Service Platform you\'ll need to extend your license.</div><a href="http://particular.net/extend-your-trial?p=servicepulse" class="btn btn-license-warning"><i class="fa fa-external-link-alt"></i> Extend your license</a><a href="#/configuration" class="btn btn-license-warning-light">View license details</a></div>';
+  '<div ><strong>Non-production development license expiring</strong><div>Your non-production development license will expire soon. To continue using the Particular Service Platform you\'ll need to extend your license.</div><a href="http://particular.net/extend-your-trial?p=servicepulse" class="btn btn-warning"><i class="fa fa-external-link-alt"></i> Extend your license</a><a href="#/configuration" class="btn btn-light">View license details</a></div>';
```
Comment (mikeminutillo) [inline src/ServicePulse.Host/vue/src/composables/serviceLicense.ts:17]: Why were these classes removed?

File: src/ServicePulse.Host/angular/app/css/particular.css [hunk 6]
```diff
@@ -2628,14 +2628,14 @@ span.fa-exclamation-triangle.danger {
...
 #toast-container > .toast-error {
-    background-image: url('../a/img/toast-danger.svg') !important;
+    background-image: url('../img/toast-danger.svg') !important;
     background-position-y: 30px;
     background-size: 28px 28px;
 }
 
 #toast-container > .toast-warning {
     background-color: #F3BC52;
-    background-image: url('../a/img/toast-warning.svg') !important;
```
Comment (mikeminutillo) [inline src/ServicePulse.Host/angular/app/css/particular.css:2638]: This is the angular css. Do we need to change this?

File: src/ServicePulse.Host/vue/package.json (hunk 2 above)
Comment (cquirosj) [reply src/ServicePulse.Host/vue/package.json:44]: Good catch. I'll take care of removing these an others in this file in the meantime

File: .gitignore (hunk 1 above)
Comment (PhilBastian) [reply .gitignore:103]: package-lock should not be ignored, i assume there was an issue on the angular side?

File: src/ServicePulse.Host/vue/src/composables/serviceLicense.ts (hunk 5 above)
Comment (soujay) [reply src/ServicePulse.Host/vue/src/composables/serviceLicense.ts:17]: @mikeminutillo - As I didn't work on these pages or the css, I am not sure. @TravisNickels are you aware of these changes?

**Review by cquirosj — APPROVED** [review]

**Pull request merged**